tests/ui/traits/next-solver/cycles/coinduction/only-one-coinductive-step-needed.current.stderr

```diff
@@ -0,0 +1,17 @@
+error[E0275]: overflow evaluating the requirement `Foo<T>: SendIndir`
+  --> $DIR/only-one-coinductive-step-needed.rs:9:15
+   |
+LL | struct Foo<T>(<Foo<T> as Trait>::Assoc);
+   |               ^^^^^^^^^^^^^^^^^^^^^^^^
+   |
+note: required for `Foo<T>` to implement `Trait`
+  --> $DIR/only-one-coinductive-step-needed.rs:18:20
+   |
+LL | impl<T: SendIndir> Trait for T {
```

Comment:
Can u explain (maybe leave a comment in the test) why this fails in the old solver?

Is it because we don't handle cycles in the projection cache correctly? Or what?